package.json

```diff
@@ -6,24 +6,33 @@
   "author": "https://teamdigitale.governo.it",
   "license": "MIT",
   "scripts": {
-    "prebuild": "rimraf dist",
+    "prebuild": "rimraf dist && npm-run-all generate:models:*",
+    "generate:models:io-functions": "rimraf generated/io-functions && mkdir -p generated/io-functions && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/io-functions/v0.64.0/api/public_api_v1.yaml --no-strict --out-dir generated/io-functions --request-types --response-decoders",
+    "generate:models:io-backend": "rimraf generated/io-backend && mkdir -p generated/io-backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/io-backend/v1.6.2/api_backend.yaml --out-dir generated/io-backend",
```

Comment:
same here, this task should not be needed, this project should not depend on `io-backend`